neondatabase/website
# feat: Inkeep chat

This pull request introduces update to AI chat feature by replacing the current custom chat implementation with InKeep integration

Steps to check:

- open [preview](https://neon-next-git-inkeep-chat-neondatabase.vercel.app/docs/introduction)
- make sure that everything looks and works as expected
